src/NuGet.Core/NuGet.Common/Logging/LoggerBase.cs

```diff
@@ -40,37 +40,37 @@ public virtual Task LogAsync(LogLevel level, string data)
             return Task.FromResult(true);
         }
 
-        public void LogDebug(string data)
+        public virtual void LogDebug(string data)
```

Comment:
No, but I think for 3rd parties this is still useful. Creating an ILogger is a pain that anyone using the APIs has to deal with.